### PR TITLE
Re-branding of help dialogs and tablet login

### PR DESCRIPTION
**Pull request by Nex-Pro**
Fixed branding on Help dialog images and Tablet login

To test:
in VR or Desktop
Open help-Controls Reference check each window says support@projectathena.io near the bottom

In VR  logout then log in. Check that there is no Highfidelity branding or links.

### CODE REVIEW

**Comment by Nex-Pro** [comment]
You can test the tablet login in desktop mode by activating the developer menu then open the developer menu-ui and uncheck "Desktop tablet becomes toolbar" Then logout and log in. Check the tablet dialog has no hifi branding.

**Comment by Nex-Pro** [comment]
Also the inner image of the loading png has been updated to show the kasen logo rather than the hifi "H"

**Comment by Nex-Pro** [comment]
Not sure where that is shown. Doh.

File: interface/resources/html/tabletHelp.html [hunk 1]
```diff
@@ -77,13 +77,14 @@
             var handControllerImageURL = null;
             var index = 0;
             var count = 3;
-			var handControllerRefURL = "https://docs.highfidelity.com/explore/get-started/vr-controls.html#vr-controls";
-			var keyboardRefURL = "https://docs.highfidelity.com/explore/get-started/desktop.html#movement-controls";
-			var gamepadRefURL = "https://docs.highfidelity.com/explore/get-started/vr-controls.html#gamepad";
+            var handControllerRefURL = "https://docs.projectathena.dev/explore/get-started/vr-controls.html#vr-controls";
+            // Link below is currently throwing a 404
+			// var keyboardRefURL = "https://docs.projectathena.dev/get-started/desktop.html#movement-controls";
+			var gamepadRefURL = "https://docs.projectathena.dev/explore/get-started/vr-controls.html#gamepad";
```
Comment (ctrlaltdavid) [inline interface/resources/html/tabletHelp.html]: URL should be "https://docs.projectathena.dev/explore/get-started/desktop.html#movement-controls" (i.e., "explore" missing in commented-out URL.

File: interface/resources/html/tabletHelp.html [hunk 2]
```diff
@@ -77,13 +77,14 @@
...
 
             function showKbm() {
                 document.getElementById("main_image").setAttribute("src", "img/tablet-help-keyboard.jpg");
-                document.getElementById("image_button").setAttribute("href", keyboardRefURL);
+                // document.getElementById("image_button").setAttribute("href", keyboardRefURL);
```
Comment (ctrlaltdavid) [inline interface/resources/html/tabletHelp.html]: Un-comment after fixing URL, above.

Comment (Nex-Pro) [reply interface/resources/html/tabletHelp.html]: I was told to remove that url by commenting it out

Comment (Nex-Pro) [reply interface/resources/html/tabletHelp.html]: Ok I see, fixing it now.

**Comment by Nex-Pro** [comment]
@ctrlaltdavid Updated

**Pull request merged**